### ISSUE TITLE
Wordlist support

### ISSUE THREAD

**Issue by ajkblue** [Lukakva/WiFicker#1]
Add support to manually input a wordlist file for password guessing.

**Comment by Lukakva** [comment]
hey. thanks for your issue. i will try and do that as soon as possible :)

**Comment by ajkblue** [comment]
@Lukakva Any progress on this?

**Comment by getsadzeg** [comment]
I don't think that wordlist is good possibility. Script with wordlist isn't WiFi hacker. There are many python scripts which take care of wordlist attacks.

I want WiFicker to add scalability support on several servers or something.

So, @ajkblue , I don't support your idea.

**Comment by Lukakva** [comment]
Good idea. It would be better if I added an option to run this script on several machines and each of them checked different permutations of given characters (or even wordlist in the future). As soon as I return to this project I'll add scalability and then wordlist.

**Comment by getsadzeg** [comment]
Scalability is very very good idea which has great chance to hack WiFi,in fact. So wordlist is old trick, It has lowest possibility to hack network.

**Comment by ajkblue** [comment]
@getsadzeg @lukakva I still believe that a wordlist is useful for faster and more common guessing of passwords, so I think that this would still be a useful feature to add in the future. But until then, can you point me to any OS X native wifi attack scripts with wordlist support?

**Comment by getsadzeg** [comment]
I have to repeat that **wordlist has lowest possibility to hack network**. And there is no need of speed if there is no quality. 

If you are interested in hacking WiFi with wordlist on OS X, take a look at the [video demonstration](https://www.youtube.com/watch?v=G6MXOzGIJZ4)
